activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/mock/MockFactory.java

```diff
@@ -176,13 +179,13 @@ public static Extensions extensions(byte[] bytes) {
         }
     }
 
-    private static Extensions getFromMap(Map<String,Object> map) throws IOException {
+    private static Map<String, Extensions> getFromMap(Map<String,Object> map) throws IOException {
```

Comment:
getFromMap? can it be serialise/deserialise or marshal/unmarshal ?

Reply:
fixed